### PR TITLE
[store] Update AppStore translations (Polish)

### PR DESCRIPTION
**Pull request by vigeron**

### CODE REVIEW

File: iphone/metadata/pl/release_notes.txt [hunk 1]
```diff
@@ -1,53 +1,54 @@
...
-• Removed bottom tab bar to show more map on the screen
-• Removed the unnecessary "route" button on the map (tap or long-tap a place on the map or select a search result to build a route using the "Route To" button instead)
-• Highlighted the Route To button and hinted the FAQ for new users
-• Fixed CarPlay dark/light mode colors
-• Fixed keyboard input on iPadOS 15+
-• "Inertial" kinetic scroll improvements
-• Removed max limit to search bookmarks
-• Brands are displayed in the Place Page
+• Nowe dane OpenStreetMap z 13 listopada 2023 r
+• Możliwe jest teraz większe przybliżenie i zobaczenie większej liczby punktów POI, w gęstych obszarach
```
Comment (ajjjjjjjj) [inline iphone/metadata/pl/release_notes.txt:2]: Teraz jest możliwe...

File: iphone/metadata/pl/release_notes.txt [hunk 2]
```diff
@@ -1,53 +1,54 @@
...
-Outdoors map style preview:
-• Tailored for out-of-the-city outdoor activities like hiking, backcountry cycling/MTB, 4x4, etc.
-• Emphasizes paths/tracks, cliffs, railways, powerlines, rivers and streams, water sources, peaks, mountain saddles and passes, waterfalls, lighthouses, caves, viewpoints, campsites, huts, shelters, picnic sites, gates, stiles, fords, etc.
-• Enable by typing `?olight` in the search field and return to the map; type `?light` to switch back to the main style
-• Use `?odark` and `?dark` for a dark style
-• Developers wanted to add a nice UI for style switching!
+Podgląd stylu mapy plenerowej:
+• Dostosowany do aktywności na świeżym powietrzu poza miastem, takich jak wędrówki, jazda na rowerze po bezdrożach/MTB, jazda terenowa 4x4, itp.
+• Podkreśla  trasy, klify, tory kolejowe, linie energetyczne, rzeki i strumienie, źródła wody, szczyty, siodła i przełęcze górskie, wodospady, latarnie morskie, jaskinie, punkty widokowe, kempingi, chatki, schronienia, miejsca na pikniki, bramy, kraty, przeprawy brodowe itp
+• Aktywuj, wpisując ?olight w pole wyszukiwania i powróć do mapy; wpisz ?light, aby wrócić do głównego stylu
```
Comment (ajjjjjjjj) [inline iphone/metadata/pl/release_notes.txt:45]: Missing quote marks here and in the line below

File: iphone/metadata/pl/release_notes.txt [hunk 3]
```diff
@@ -1,53 +1,54 @@
...
+• Użyj ?odark i ?dark dla ciemnego stylu
+• Poszukiwani deweloperzy Androida i iOS do pomocy w dodaniu przyjaznego interfejsu użytkownika do zmiany stylu!
 
-Translations
-• Improved "avoid …" routing options wording
-• Improved Arabic, Basque, Chinese, Czech, Danish, Finnish, Greek, Hungarian, Indonesian, Italian, Korean, Marathi, Polish, Romanian, Slovak, Spanish, Sweden, Thai, Turkish, Ukrainian, and Vietnamese translations
+Tłumaczenia
+• Poprawiono sformułowanie opcji routingu „unikaj…”
```
Comment (ajjjjjjjj) [inline iphone/metadata/pl/release_notes.txt:50]: routingu = wyznaczania trasy

File: iphone/metadata/pl/release_notes.txt [hunk 4]
```diff
@@ -1,53 +1,54 @@
...
+• Poprawiono tłumaczenia Arabic, Basque, Chinese, Czech, Danish, Finnish, Greek, Hungarian, Indonesian, Italian, Korean, Marathi, Polish, Romanian, Slovak, Spanish, Sweden, Thai, Turkish, Ukrainian, i Vietnamese
```
Comment (ajjjjjjjj) [inline iphone/metadata/pl/release_notes.txt:51]: redundant coma before last language

File: iphone/metadata/pl/release_notes.txt (hunk 2 above)
Comment (vigeron) [reply iphone/metadata/pl/release_notes.txt:45]: they are not in the original text

**Review by ajjjjjjjj — APPROVED** [review]

**Review by biodranik — APPROVED** [review]
Thanks! Unfortunately, we can not update translations in the AppStore after publishing the release.

**Pull request merged**